batch/batch.go

```diff
@@ -1355,8 +1360,12 @@ func (b *Batch) makeFragments(frags, clearFrags fragments) (fragments, fragments
 
 			if col/shardWidth != curShard {
 				curShard = col / shardWidth
+				// the API treats "" as standard
 				curBM = frags.GetOrCreate(curShard, field.Name, "")
 				clearBM = clearFrags.GetOrCreate(curShard, field.Name, "")
+				if opts.TrackExistence {
+					existCurBM = frags.GetOrCreate(curShard, field.Name, "existence")
```

Comment:
yeah. we're also using "standard" in some but not all places in batch, and in other places we're using "" and assuming it will be converted *either* to "standard" or to the bsig field name, and i distrust that.